packages/integration-tests/src/test-project-setup/data_storage_auth_with_triggers.ts

```diff
@@ -460,4 +462,100 @@ class DataStorageAuthWithTriggerTestProject extends TestProjectBase {
       throw err;
     }
   };
+
+  private createSQSQueue = async () => {
+    const createQueueResponse = await this.sqsClient.send(
+      new CreateQueueCommand({ QueueName: this.queueName })
+    );
+    this.queueUrl = createQueueResponse.QueueUrl ?? '';
+  };
+
+  private deleteSQSQueue = async () => {
+    if (this.queueUrl) {
+      await this.sqsClient.send(
+        new DeleteQueueCommand({
+          QueueUrl: this.queueUrl,
+        })
+      );
+    }
+  };
+
+  private addLambdaPermissionForSQS = async (backendId: BackendIdentifier) => {
+    this.funcWithSchedulePhysicalIds =
+      await this.resourceFinder.findByBackendIdentifier(
+        backendId,
+        'AWS::Lambda::Function',
+        (name) => name.includes('funcWithSchedule')
+      );
+    const getQueueAttributesResponse = await this.getSQSQueueAttributes(
+      this.queueUrl
+    );
+    const queueArn = getQueueAttributesResponse.Attributes?.QueueArn ?? '';
+
+    const sqsPolicy = new PolicyDocument({
+      statements: [
+        new PolicyStatement({
+          actions: ['sqs:SendMessage'],
+          effect: Effect.ALLOW,
+          resources: [queueArn],
+        }),
+      ],
+    });
```

Comment:
Should we create the SQS queue as part of the Backend App itself? This way a lot of the extra work will be handled with the CDK/CFN rather than managing it here.

Reply:
There's one more benefit of nesting SQS under backend app. We won't have to include SQS in cleanup script as orphan stack deletion will take care of it in case test crashes and leaves it behind.